src/Umbraco.Web.UI.Client/src/less/components/overlays.less

```diff
@@ -249,6 +241,24 @@
     font-size: 12px;
 }
 
+.umb-overlay.umb-overlay-target,
+.umb-overlay.umb-overlay-left,
+.umb-overlay.umb-overlay-right {
```

Comment:
The overlay target was the "old" overlay shown at target position when clicking e.g. pickers to select allowed children on a content type, but most of these have been replaced with editor service.
In fact I think only core may used this in legacy Grid layout editor.